# NOTE: You have to change this node name if you change the node name in the launchfile.
republisher:
  ros__parameters:
    # The name of the topics to republish from
    from_topics:
      - /camera/color/image_raw/compressed
      - /camera/color/camera_info
      - /camera/aligned_depth_to_color/image_raw
      - /camera/aligned_depth_to_color/camera_info

    # The types of topics to republish from
    topic_types:
      - sensor_msgs.msg.CompressedImage
      - sensor_msgs.msg.CameraInfo
      - sensor_msgs.msg.Image
      - sensor_msgs.msg.CameraInfo

    # The name of a post-processing function to apply to the message before
    # republishing it. Current options are "none" or "mask", and "mask"
    # only applies to messages of type "Image" "mask" loads the binary mask in
    # `mask_relative_path` and applies it to the Image.
    post_processors:
      - none
      - none
      - mask
      - none

    # The binary mask used for "mask" post-processing. This mask will get scaled,
    # possibly disproportionately, to the same of the image.
    mask_relative_path: model/fork_handle_mask.png